Library/Formula/nspr.rb

```diff
@@ -32,12 +34,12 @@ def install
         --enable-strip
         --enable-pthreads
         --enable-ipv6
-        --enable-macos-target=#{MacOS.version}
       ]
-      args << "--enable-64bit" if MacOS.prefer_64_bit?
+      args << "--enable-macos-target=#{MacOS.version}" if OS.mac?
+      args << "--enable-64bit" if MacOS.prefer_64_bit? || OS.linux?
```

Comment:
I'm guessing based on the changes that were made to the sonar formula that you probably don't want the `OS.linux?` bit here.

I'll happily modify it tomorrow if you'd like, but I'm tired so I'm going to leave it for now.